.npmrc

```diff
@@ -0,0 +1,3 @@
+node-linker=hoisted
+public-hoist-pattern=*
+shamefully-hoist=true
```

Comment:
As we're using pnpm for this project, is this needed?

Reply:
It was needed for me to run "pnpm package", but I will remove it from the public repository